Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -195,21 +197,33 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
 
         switch (this._type) {
         case WI.InlineSwatch.Type.Color:
+            let title = this._swatchTooltip;
             if (!this._readOnly) {
-                let title = WI.UIString("Click to select a color.");
+                title = WI.UIString("Click to select a color.");
```

Comment:
Completely, it's specifically for the swatches that the user clicks on, because we want to display only the name of said swatches's variable on the tool tip

Reply:
Also, what did you mean by the previous comment ?